self-hosted/configuration/timescaledb-config.md

```diff
@@ -21,6 +21,21 @@ Max background worker processes allocated to TimescaleDB. Set to at least 1 +
 the number of databases loaded with a TimescaleDB extension in a PostgreSQL
 instance. Default value is 16.
 
+## Query Planning and Execution
+
+### `timescaledb.enable_chunkwise_aggregation (bool)`
+If enabled, aggregations are converted into partial aggregations during query
+planning. The first part of the aggregation is executed on a per-chunk basis.
+Then, these partial results are combined and finalized. Splitting aggregations
+decreases the size of the created hash tables and increases data locality, which
+speeds up queries.
+ 
+
+### `timescaledb.vectorized_aggregation (bool)`
```

Comment:
Will be renamed in https://github.com/timescale/timescaledb/pull/6372 to follow the naming pattern  for GUCs